commandapi-platforms/commandapi-bukkit/commandapi-bukkit-vh/src/main/java/dev/jorel/commandapi/CommandAPIVersionHandler.java

```diff
@@ -68,9 +70,54 @@ public interface CommandAPIVersionHandler {
 				case "1.20.3", "1.20.4" -> new NMS_1_20_R3();
 				case "1.20.5", "1.20.6" -> new NMS_1_20_R4();
 				case "1.21", "1.21.1" -> new NMS_1_21_R1();
-				default -> throw new UnsupportedVersionException(version);
+				default -> null;
 			};
+			if (platform != null) {
+				return platform;
+			} else {
+				if (CommandAPI.getConfiguration().shouldBeMoreLenientForMinorVersions()) {
+					return minecraftVersion.matchesPatch((NMS<?>) latestNMS);
```

Comment:
The cast to `NMS<?>` here could be avoided by declaring `latestNMS` with type `CommandAPIBukkit<?>` rather than `CommandAPIPlatform<?, ?, ?>`, since `CommandAPIBukkit<?>` implements both of those.